Dockerfile-dind-rootless.template

```diff
@@ -35,6 +36,37 @@ RUN set -eux; \
 	; \
 	rm rootless.tgz; \
 	\
+	echo '{{
+		{
+			name: "docker",
+			version: .version,
+			params: {
+				os_name: "alpine",
+				os_version: .alpine
+			},
+			licenses: [
+				"Apache-2.0"
+			]
+		} | sbom | tostring
+	}}' > /usr/local/bin/docker.spdx.json; \
+	echo '{{
+		{
+			name: "docker-compose",
+			version: .version,
+			params: {
+				os_name: "alpine",
+				os_version: .alpine
+			},
+			licenses: [
+				"Apache-2.0"
+			]
+		} | sbom | tostring
+	}}' > /usr/local/bin/docker-compose.spdx.json; \
+	\
+	dockerd --version; \
+	containerd --version; \
+	ctr --version; \
+	runc --version; \
```

Comment:
These binaries all come from the `dind` image that this is from; they are already tested when downloaded rather than in a dependent image to make sure a broken binary doesn't get committed to an image.